validmind/ai.py

```diff
@@ -71,7 +71,7 @@ def __get_client_and_model():
 
     if "OPENAI_API_KEY" in os.environ:
         __client = OpenAI(api_key=os.environ.get("OPENAI_API_KEY"))
-        __model = os.environ.get("VM_OPENAI_MODEL", "gpt-4-turbo-preview")
```

Comment:
We can address this later, but it might be worth considering giving users the option to choose their model, with GPT-4 as the default.

Reply:
Yep thats a good idea... I think ideally we move to using our own API instead of calling the LLM directly from the dev framework but I am open to ideas there. This was more for a temporary solution AFAIK.